BoundedKnapsack/Hints.qs

```diff
@@ -0,0 +1,96 @@
+﻿//////////////////////////////////////////////////////////////////////
+// This file contains hints to all tasks.
+// The tasks themselves can be found in Tasks.qs file.
+// We recommend that you try to solve the tasks yourself first,
+// but feel free to look up these hints, and ultimately
+// the solutions if you get stuck.
+//////////////////////////////////////////////////////////////////////
+
+// Hints have been provided to clarify and guide your work through some of
+// the harder or more complicated katas. The hints within each task gradually reveal
+// more and more of the solution. You might consider reading hints for a task
+// up to a point of understanding, and then try finishing the task yourself.
+
+
+
+
+// Task 1.2. Calculate Total Weight or Profit
+//
+//	1) The IncrementByInteger operation may be of use: https://docs.microsoft.com/en-us/qsharp/api/qsharp/microsoft.quantum.arithmetic.incrementbyinteger

```

Comment:
```suggestion
//  1) The IncrementByInteger operation may be of use: https://docs.microsoft.com/en-us/qsharp/api/qsharp/microsoft.quantum.arithmetic.incrementbyinteger
```

I noticed that there are tabs in this file. Could you please remove all tabs in one of the next commits?